wiki/Beatmaps/en.md

```diff
@@ -1,78 +1,67 @@
+Pending/WIP beatmaps use the question mark icon (![Question mark icon](/wiki/shared/status/pending.png)) in the song selection screen.
 
 ### Qualified
 
-Qualified beatmaps use the check icon (![Check icon](/wiki/shared/status/qualified.png)) in the song select screen.
+Qualified beatmaps are in the last stage of the [Beatmap ranking procedure](/wiki/Beatmap_ranking_procedure). These beatmaps have been nominated by at least two [Beatmap Nominators](/wiki/Beatmap_Nominators) who have made sure that the beatmap meets the standards of the [Ranking Criteria](/wiki/ranking_criteria).
+
+Beatmaps stay qualified for a period of approximately 7 days before they enter the [Ranked]((#ranked)) section permanently and become endorsed as official content. This period serves as a last line of defense in which any flaws and mistakes that might have slipped through can be fixed. In this period, beatmaps can be disqualified, changed and re-qualified without reseting the time it takes for the beatmap to become ranked. The Qualified section is in that sense, the last step before being considered a fully fledged beatmap.
 
-These beatmaps are in the last stage of the beatmapping and modding process. Before the beatmap begins playing, a dialogue will display, noting the status of the beatmap will cause:
+Qualified beatmaps have leaderboards. Though, upon playing the beatmap a dialogue will display, noting that:
 
 - No [performance points (pp)](/wiki/Performance_Points) will be awarded.
 - All scores will be deleted when the status is changed.
-  - If the status is changed to ranked, approved, or back to pending; all scores will be wiped!
 
-When a beatmap is qualified, it has passed the [ranking criteria](/wiki/ranking_criteria) and has been checked by two [Beatmap Nominators](/wiki/Beatmap_Nominators). Qualified beatmaps is the last line of defense as they are required to be in this status for one week before becoming [ranked](#ranked) or [approved](#approved) permanently.
+Qualified beatmaps use the check icon (![Check icon](/wiki/shared/status/qualified.png)) in the song selection screen.
+
+### Ranked
+
+Ranked beatmaps are endorsed as official content, in the sense that they have reached or surpassed the standards of the [Ranking Criteria](/wiki/ranking_criteria). This is to ensure that most players can expect a certain kind of standard while playing them. Ranked beatmaps allow other players to compete on leaderboards and to gain performance points from setting scores on them. Once a beatmap has achieved Ranked status, its status cannot be altered again (except for extraordinary cases).
```

Comment:
```suggestion
Ranked beatmaps are endorsed as official content, in the sense that they have reached or surpassed the standards of the [Ranking Criteria](/wiki/ranking_criteria). This is to ensure that most players can expect a certain kind of standard while playing them. Ranked beatmaps allow other players to compete on leaderboards and to gain performance points from setting scores on them. Once a beatmap has achieved ranked status, its status cannot be altered again (except for extraordinary cases).
```
I lower-cased the "r" in "Ranked status" because I think it's talking about a status and not the category.